lib/middleman-simple-thumbnailer/image.rb

```diff
@@ -89,7 +94,7 @@ def cached_resized_img_abs_path
     end
 
     def cached_thumbnail_available?
-      File.exist?(cached_resized_img_abs_path)
+      File.exist?(cached_resized_img_abs_path) && (File.mtime(cached_resized_img_abs_path) >= File.mtime(abs_path))
```

Comment:
No I did not encounter any problem. It was also "better safe than sorry" thinking. So this might not be necessary.